src/v/raft/state_machine_manager.h

```diff
@@ -220,6 +256,7 @@ class state_machine_manager final {
     ss::abort_source _as;
     ss::scheduling_group _apply_sg;
     snapshot_at_offset_supported _supports_snapshot_at_offset{true};
+    storage::simple_snapshot_manager _snapshot_mgr;
```

Comment:
Nit: a more specific variable name would be nice. Or is the idea to maybe put other things into this snapshot in the future?